crates/client-api/src/routes/subscribe.rs

```diff
@@ -43,7 +43,12 @@ pub struct SubscribeParams {
 #[derive(Deserialize)]
 pub struct SubscribeQueryParams {
     pub client_address: Option<AddressForUrl>,
-    pub compression: Option<Compression>,
+    #[serde(default)]
+    pub compression: Compression,
+    /// Whether we want "light" responses, tailored to network bandwidth constrained clients.
+    /// This knob works by setting other, more specifc, knobs to the value.
+    #[serde(default)]
+    pub light: bool,
```

Comment:
I thought the plan was to encode this as part of the protocol? Something like `v1.bin.spacetimedb.light` or am I misremembering? I could definitely be misremembering.

Reply:
Ack. It's definitely a bikeshed. Just wondering if you knew what we had originally said.